example/randomCells2D/src/randomCells2D.cpp

```diff
@@ -201,16 +201,16 @@ auto main() -> int
     RandomEngineVector<Acc>* const ptrBufAccRandV{alpaka::getPtrNative(bufAccRandV)};
 
     InitRandomKernel initRandomKernel;
-    auto pitchBufAccRandS = alpaka::getPitchBytes<1u>(bufAccRandS);
+    auto pitchBufAccRandS = alpaka::getPitchesInBytes(bufAccRandS)[1];
```

Comment:
`bufAcccRandS` is a 2-diensional buffer therefore this should be `auto pitchBufAccRandS = alpaka::getPitchesInBytes(bufAccRandS)[0];`